modules/op-installing-tkn-on-linux.adoc

```diff
@@ -12,11 +12,11 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
 
 . Download the relevant CLI.
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.17.2/tkn-linux-amd64-0.17.2.tar.gz[Linux (x86_64, amd64)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.19/tkn-linux-amd64-0.19.tar.gz[Linux (x86_64, amd64)]
```

Comment:
Due to a regression bug fix the version of `tkn` got changed from `0.19` to `0.19.1`. Links needs to be updated